src/Cassandra.IntegrationTests/TestBase/TestCassandraVersion.cs

```diff
@@ -105,7 +105,7 @@ public static bool VersionMatch(Version expectedVersion, bool requiresDse, bool
 
             if (TestClusterManager.IsScylla && requiresOss)
             {
-                message = "Test designed to run with OSS Cassandra (executing Scylla)";
+                message = null;
```

Comment:
Do you really need this change ?

Reply:
Yeah, but I have created an issue for that: https://github.com/scylladb/csharp-driver/issues/20